app/controllers/trips_controller.rb

```diff
@@ -12,6 +12,27 @@ def dashboard
     authorize Trip
   end
 
+  def import
+    authorize Trip
+    if request.post?
+      @wrong_ids = []
+      @osies = trip_params[:students_to_import].split(" ").map(&:to_i)
+      for i in @osies
+        if Student.where(osis: i).exists?
+          if !@trip.students.where(osis: i).exists?
```

Comment:
Something has happened here to the indentation

Reply:
My suggestion for checking for wrong IDs is to take advantage of the fact that `find_by` returns `nil` if no record is found.

So you can get rid of the if condition `Student.where(osis: i).exists?`